<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="text_output">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>751</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: black; color: #39ff14; font-family: Monospace; font-size: 12px;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>186</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btn_monitor">
       <property name="text">
        <string>ğŸ–¥ï¸ System Monitor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_disk">
       <property name="text">
        <string>ğŸ’¾ Disk Usage</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_fim_init">
       <property name="text">
        <string>ğŸ”’ FIM: Create Baseline</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_fim_check">
       <property name="text">
        <string>ğŸ•µï¸â€â™‚ï¸ FIM: Check Integrity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_logs">
       <property name="text">
        <string>ğŸ›¡ï¸ Log Analyzer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_backup">
       <property name="text">
        <string>ğŸ“¦ Automated Backup</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_battery">
       <property name="text">
        <string>ğŸ”‹Battery Health</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>993</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
